challenge-two/src/agent.spec.ts

```diff
@@ -0,0 +1,172 @@
+import {
+  Finding,
+  HandleTransaction,
+  FindingSeverity,
+  FindingType,
+} from "forta-agent";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { ethers } from "ethers";
+import { provideHandleTransaction } from "./agent";
+import { SWAP_EVENT_SIGNATURE } from "./constants";
+
+describe("Uniswap V3 Pool Swap Detection", () => {
+  let handleTransaction: HandleTransaction;
+
+  const MOCK_POOL_ADDRESS_1 =
+    "0x3ebFBF0CDaeF17aae3598Aa9A6eaC55Ed74276D0".toLocaleLowerCase();
```

Comment:
```suggestion
    "0x3ebFBF0CDaeF17aae3598Aa9A6eaC55Ed74276D0".toLowerCase();
```
`.toLowerCase()` should be sufficient.